lib/posix/Kconfig.signal

```diff
@@ -24,3 +18,12 @@ config POSIX_SIGNAL_STRING_DESC
 	  Will use 256 bytes of ROM.
 
 endif
+
+# needed outside of if clause above to define constants & types in signal.h
+config POSIX_RTSIG_MAX
+	int "Maximum number of realtime signals"
+	default 31 if POSIX_SIGNAL
+	default 0 if !POSIX_SIGNAL
```

Comment:
probably don't need the if !POSIX_SIGNAL here?

Reply:
fixed!